<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CondJumpWidget</class>
 <widget class="QWidget" name="CondJumpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>331</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>输出设置</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>32</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>跳转类型</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK跳转</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOKJump">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NG跳转</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxNGJump">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>280</y>
      <width>311</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>IO使能设置</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>291</width>
       <height>103</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>OK IO 输出</string>
     </property>
     <widget class="QRadioButton" name="radioButtonOKIO1">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>33</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonOKIO2">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>33</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonOKIO3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>66</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonOKIO4">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>66</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO4</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>240</y>
       <width>291</width>
       <height>103</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>NG IO 输出</string>
     </property>
     <widget class="QRadioButton" name="radioButtonNGIO1">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>33</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonNGIO2">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>33</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonNGIO3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>66</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonNGIO4">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>66</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IO4</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxNGIO">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>216</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>NG IO 使能</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOKIO">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>35</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK IO 使能</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>631</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>跳转设置</string>
   </property>
   <widget class="QListWidget" name="listWidgetAssociationStepsData">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>108</y>
      <width>501</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>29</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>27</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关联步骤</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbAssociationSteps">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>30</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAndAll">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>74</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>全与</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNotAll">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>74</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>全或</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>72</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>73</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>跳转关联步骤列表</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
